skfuzzy/control/visualization.py

```diff
@@ -48,21 +56,27 @@ def view(self, *args, **kwargs):
                     self.fuzzy_var.universe, zeros, cut_mfs[label],
                     facecolor=color, alpha=0.4)
 
-        # Plot defuzzified output if available
-        if len(cut_mfs) > 0:
-            crip_value = defuzz(self.fuzzy_var.universe, output_mf,
-                                self.fuzzy_var.defuzzify_method)
-            if crip_value is not None:
+        # Plot crisp value if available
+        if len(cut_mfs) > 0 and not all(output_mf == 0):
+            crisp_value = None
+            if hasattr(self.fuzzy_var, 'input'):
+                crisp_value = self.fuzzy_var.input[sim]
+            elif hasattr(self.fuzzy_var, 'output'):
+                crisp_value = self.fuzzy_var.output[sim]
+
+            if crisp_value is not None:
                 y = interp_membership(self.fuzzy_var.universe,
-                                      output_mf, crip_value)
-                self.ax.plot([crip_value] * 2, [0, y],
+                                      output_mf, crisp_value)
+                if y < 0.1:
+                    y = 1.
```

Comment:
This really worried me for a moment, then I got it. Small lines don't show.

Made me think though: would it make sense to invert the line if membership is below 0.5? A vertical line starting from the top, going down to the membership value instead of up from the bottom? Or do you think that would confuse more than it would clarify?